### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kyscott18**

### CODE REVIEW

**Review by roninjin10 — APPROVED** [review]

**Comment by holgerd77** [comment]
@roninjin10 Hi there, no big deal and in this case totally unproblematic 🙂, but we have a security policy that only team members should approve and/or merge PRs. Will keep your permissions as is, but please do not do approvals on other PRs. Thanks for your understanding! 🙂 🙏

**Comment by roninjin10** [comment]
Hey sorry about that I actually meant to dismiss my approval but got distracted

**Pull request merged**
